newsletters/templates/newsletters/create.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'tgrsite/main.html' %}
+{% block title %}
+New Newsletter
+{% endblock %}
+{% block body %}
+
+<h1>New Newsletter</h1>
+<form action="" method="POST">
+	{% csrf_token %}
+	{{ form.non_field.errors }}
```

Comment:
-> `non_field_errors`

Reply:
I copied that from one of your forms, so I'll find that form and fix it as well. :stuck_out_tongue: 
Probably same source as `action=""`.

Reply:
Fixed.